README.md

```diff
@@ -181,13 +254,27 @@ coreApi.apiConfig().paymentOverrideNotification("https://example.com/test1,https
 ```
 When both `X-Append-Notification` and `X-Override-Notification` are used together then only override will be used.
 
+### Handling Error / Exception
+When using function that result in Midtrans API call e.g: CoreApi.chargeTransaction(...) or SnapApi.createTransaction(...) there's a chance it may throw error (MidtransError object), the error object will contains below properties that can be used as information to your error handling logic:
+```java
+try {
+    JSONObject result = CoreApi.chargeTransaction(param);
+} catch (MidtransError e) {
+    e.printStackTrace();
+    e.getStatusCode(); // basic error message string
+    e.getMessage(); // HTTP status code e.g: 400, 401, etc.
+    e.getResponseBody(); // API response body
+    e.getResponse(); // raw OkHttp response object
+}
+```
 #### CoreAPI Simple Sample Usage
 ```java
 
 import com.midtrans.Config;
 import com.midtrans.ConfigFactory;
-import com.midtrans.service.MidtransCoreApi;
+import com.midtrans.httpclient.CoreApi;
 import com.midtrans.httpclient.error.MidtransError;
+import com.midtrans.Midtrans;
```

Comment:
Please check if import is still used, looks like Config & ConfigFactory is no longer used in the sample code? Maybe remove from import.